src/EditorFeatures/Core/FindUsages/IDefinitionsAndReferencesFactory.cs

```diff
@@ -241,10 +245,13 @@ internal static class DefinitionItemExtensions
 
             var document = referenceLocation.Document;
             var sourceSpan = location.SourceSpan;
-            var options = ClassificationOptions.From(document.Project);
+
+            // TODO:
+            // var options = await context.GetOptionsAsync(document.Project.Language, cancellationToken).ConfigureAwait(false);
```

Comment:
can do in this pr?

Reply:
It would require moving the options to global options which affects a lot of other code.